src/patternfly/components/Page/page-header-brand-link.hbs

```diff
@@ -1,4 +1,4 @@
-<a class="pf-c-page__header-brand-link{{#if page-header-brand-link--modifier}} {{page-header-brand-link--modifier}}{{/if}}"
+<a href="{{page-header-brand-link--href}}" class="pf-c-page__header-brand-link{{#if page-header-brand-link--modifier}} {{page-header-brand-link--modifier}}{{/if}}"
```

Comment:
```suggestion
<a href="{{#if page-header-brand-link--href}}{{page-header-brand-link--href}}{{else}}#{{/if}}{{page-header-brand-link--href}}" class="pf-c-page__header-brand-link{{#if page-header-brand-link--modifier}} {{page-header-brand-link--modifier}}{{/if}}"
```

Reply:
What do you think about making `#` the default and allowing it to be updated if needed?

Reply:
Thats a good idea

Reply:
Should we also add a condition for if there is no link, then instead of an `<a>` its a `<span>`